keep-ui/features/workflows/builder/ui/Editor/TriggerEditor.tsx

```diff
@@ -100,30 +107,55 @@ export function TriggerEditor() {
               </Button>
             </div>
             {properties.alert &&
-              Object.keys(properties.alert ?? {}).map((filter) => (
-                <div key={filter}>
-                  <Subtitle className="mt-2.5">{filter}</Subtitle>
-                  <div className="flex items-center mt-1">
-                    <TextInput
-                      key={filter}
-                      placeholder={`Set alert ${filter}`}
-                      onChange={(e: any) =>
-                        updateAlertFilter(filter, e.target.value)
-                      }
-                      value={
-                        (properties.alert as any)[filter] || ("" as string)
-                      }
-                    />
-                    <Icon
-                      icon={BackspaceIcon}
-                      className="cursor-pointer"
-                      color="red"
-                      tooltip={`Remove ${filter} filter`}
-                      onClick={() => deleteFilter(filter)}
-                    />
+              Object.keys(properties.alert ?? {}).map((filter) =>
```

Comment:
I think we should place CEL input above deprecated filters and add some indication of deprecation, something like this

<img width="1552" alt="Screenshot 2025-04-29 at 18 23 08" src="https://github.com/user-attachments/assets/99839e54-e358-4862-8da7-07b1eb65fbe6" />